### ISSUE TITLE
Add help page.

### ISSUE THREAD

**Issue by atm1504** [dsciitpatna/Shabd#150]
Add help page which describes the features of the app. Add it in the profile section.
Enable speak feature in the help page.

**Comment by archanas10** [comment]
Hey, 
I'm a little new here, and looking forward to contribute something, if you dont mind. Is this issue still open?

**Comment by atm1504** [comment]
@SAS1998 yes, it's open. You can take it up. Please join the gitter channel for communication

**Comment by Tushar98644** [comment]
Assign me @atm1504

**Comment by Diptadip** [comment]
> Assign me @atm1504

Assigned
